docs/source/aws/usage.rst

```diff
@@ -31,7 +31,22 @@ Additionally some filters and actions may generate their own metrics.
 In order to enable metrics output, the boolean metrics
 flag needs to be specified when running Cloud Custodian::
 
-  $ custodian run -s <output_directory> --metrics <policyfile>.yml
+  $ custodian run -s <output_directory> --metrics-enabled aws <policyfile>.yml
+
+You can also consolidate metrics into a single account by specifying the ``master``
+location in the cli::
+
+  $ custodian run -s <output_directory> --metrics-enabled aws://master
+
+Additionally, to use a different namespace other than the default ``CloudMaid``, you can
+add the following query parameter to the metrics flag::
+
+  $ custodian run -s <output_directory> --metrics-enabled aws://?namespace=foo
```

Comment:
should also cover region aggregation, ie. ?region=us-east-1 

also master aggregation only makes sense if doing --assume or c7n-org in a single account context, there is no master, ie master=current account.

its also worth covering that the dimensions change if using region or master to include enough context to disambiguate between accounts and regions (ie those are additional dimensions).